.github/dependabot.yml

```diff
@@ -1,14 +1,32 @@
 version: 2
 updates:
 
+  # Maintain dependencies for Python
+  - package-ecosystem: "pip"
+    directory: "/docs"
+    schedule:
+      interval: "weekly"
+      day: "friday"
+      time: "05:00"
+      timezone: "Europe/Amsterdam"
+    open-pull-requests-limit: 20
```

Comment:
I have to say that this check scares me. It has been a while... :D

Reply:
Yes, but luckily we don't have that many dependencies ([only 3](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/chore/set_dependabot_weekly/docs/requirements.txt)) and we have been keeping them relatively up to date ([see latest PR](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/688)). But let's see...